spec/index.adoc

```diff
@@ -2509,9 +2509,16 @@ Conditions::
     ) ∨ (
       E.content.canister_id ≠ ic_principal
       S.canisters[E.content.canister_id] ≠ EmptyCanister
-      Arg = { data = E.content.arg; caller = E.content.sender; method = E.content.method_name; time = S.time[E.content.canister_id] }
+      Arg = { data = E.content.arg; caller = E.content.sender }
```

Comment:
Why is `caller` still part of `Arg`?

Reply:
I don’t remember :-). Would you mind taking over? I think you are currently deeper into this than I am

Reply:
I'd be happy to take over. :) Thank you for your trust!

Reply:
I've turned `Arg` to `Blob` and have `CallerId = Principal` passed separately.